conans/client/packager.py

```diff
@@ -82,7 +82,8 @@ def recipe_has(attribute):
         raise ConanException(e)
 
     _create_aux_files(install_folder, package_folder, conanfile, copy_info)
-    output.success("Package '%s' created" % os.path.basename(package_folder))
+    pkg_id = pkg_id or os.path.basename(package_folder)
```

Comment:
Just print here the pkg_id, let be the caller the one that defines its correct value